rust/connlib/libs/client/src/control.rs

```diff
@@ -260,7 +261,7 @@ impl<CB: Callbacks + 'static> ControlPlane<CB> {
 impl<CB: Callbacks + 'static> ControlSession<Messages, CB> for ControlPlane<CB> {
     #[tracing::instrument(level = "trace", skip(private_key, callbacks))]
     async fn start(
-        private_key: StaticSecret,
+        private_key: Secret<StaticSecret>,
```

Comment:
`StaticSecret` already implements `Zeroize` and doesn't implement debug/display so no need to wrap it.